src/oemof/solph/processing.py

```diff
@@ -369,6 +369,8 @@ def remove_nones(com):
             if len(value) == 0 or value[0] is None:
                 del com["sequences"][ckey]
 
+    exclude_attrs = exclude_attrs or []
```

Comment:
In my opinion, a classical
```suggestion
    if exclude_attrs is None:
        exclude_attrs = []
```
would be more comprehensible.

Reply:
okay for me - I like this style, but adapted your suggestion (for both places)